Profesja/Test/Profesja_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include "Profesja.hpp"
```

Comment:
testów nie przeglądam i nie będę robił ich review. Sami musicie o nie zadbać, takie jest moje podejście.


Dodatkowo do zastosowania wszędzie:
- pliki nazwać tak jak klasa w nich się nazywa
- jeden plik = jednak klasa
- wyrównać spacje. tabulatory --> jak zobaczę złe formatowanie w kodzie, który będzie w bazie to robię `git blame` i odejmuje punkty osobie, która to scommitowała.